SampleApp/SampleApp/SampleFailingView.swift

```diff
@@ -0,0 +1,27 @@
+// © 2016 LinkedIn Corp. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+
+public class SampleFailingView: UIView {
+
+    private static let nibName = "SampleFailingView"
+    
+    @IBOutlet weak var label: UILabel!
+    @IBOutlet weak var button: UIButton!
+    
+    class func loadFromNib() -> SampleFailingView {
+        return NSBundle.mainBundle().loadNibNamed(SampleFailingView.nibName, owner: nil, options: nil)[0] as! SampleFailingView
+    }
+
+    func setup(json: [NSObject: AnyObject]) {
+        label.text = json["text"] as? String
+        button.titleLabel?.text = json["buttonText"] as? String
```

Comment:
Use button.setTitle(json["buttonText"] as? String, forState: .Normal) instead.

This code currently does nothing (I don't think that's intended). The above code produces much better screenshots.